boards/radxa-RockPi4/preboot-usb-start.patch

```diff
@@ -0,0 +1,27 @@
+Patch-Source: https://www.mail-archive.com/u-boot@lists.denx.de/msg371891.html
+
+This patch fixes a problem with USB keyboard not working in the boot menu.
+I tested it on ROCK Pi 4 B+ and it works, but the keyboard must be connected
+to one of the *blue* USB ports.
+--
+From: Marcin Juszkiewicz <marcin@juszkiewicz.com.pl>
+Date: Wed, 3 Jan 2020 12:45:26 +0700
+Subject: [PATCH] rockchip: rockpro64: initialize USB in preboot
```

Comment:
sneaky!

I found "this patch", and got confused seeing it was accepted and merged in 2020.

Note to anyone missing the sneaky bits: this is an edited patch for another board.

We probably should instead move this to Tow-Boot config, since as-this-is it won't be inherited by the 4C, since it uses a different defconfig.

(Still notes to me)